src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -116,7 +116,8 @@ class AuthScreens extends React.Component {
         PersonalDetails.fetchPersonalDetails();
         User.getUserDetails();
         User.getBetas();
-        fetchAllReports(true, true);
+        PersonalDetails.fetchCurrencyPreferences();
+        fetchAllReports(true, true, true);
```

Comment:
Extra `true` added here as an argument.

Reply:
I've resolved this as part of a PR that [touches the currency logic.](https://github.com/Expensify/Expensify.cash/pull/3514#issue-666950596)